ios/App/App/AppDelegate.swift

```diff
@@ -0,0 +1,74 @@
+import UIKit
+import Capacitor
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+  var window: UIWindow?
+
+
```

Comment:
Codacy found an issue: [Limit vertical whitespace to a single empty line. Currently 2.](https://app.codacy.com/gh/numbersprotocol/capture-lite/pullRequest?prid=6209019)